mitosheet/src/jupyterRender.tsx

```diff
@@ -3,16 +3,20 @@ import ReactDOM from 'react-dom';
 import Mito from './components/Mito';
 import { getAnalysisDataFromString, getSheetDataArrayFromString, getUserProfileFromString } from './jupyter/jupyterUtils';
 
-// We replace all of the following REPLACE_THIS_WITH... variables with 
-// the actual values they should be upon creation of the mitosheet. This is
-// how we get data needed at initialization from the mito backend to the 
-// frontend
+// We replace the following byte arrays with the real byte arrays of the utf8 encoded
+// JSON for the sheet data array, etc. We pass this encoded because the JSON parsing
+// when we don't gets really complicated trying to replace \t, etc
+const sheetDataBytes = new Uint8Array([]);
+const analysisDataBytes = new Uint8Array([]);
+const userProfileBytes = new Uint8Array([]);
```

Comment:
Just to make this more clear, could we add a comment that says like "Do not make any edits to the following three lines since we are searching for this exact text in the mito_backend to replace it with the actual data if it exists.